components/Themes/.jazzy.yaml

```diff
@@ -0,0 +1,5 @@
+# Auto-generated by scripts/generate_jazzy_yamls.sh. Used primarily by scripts/external/arc-jazzy-linter.
+module: FeatureHighlight
```

Comment:
This seems suspicious.
Is this a C&P error?
Can you rerun scripts/generate_jazzy_yamls.sh?